litellm/__init__.py

```diff
@@ -78,7 +78,7 @@ def identify(event_details):
 ####### Secret Manager #####################
 secret_manager_client = None
 ####### COMPLETION MODELS ###################
-open_ai_chat_completion_models = [
+open_ai_chat_completion_models: str = [
```

Comment:
@WilliamEspegren fyi this failed our linting - this is a list

Reply:
😬ooopsie, great catch by you @ishaan-jaff